cmdriver/cmdriver.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Service
```

Comment:
Why is this here?  It suffices to annotate the Pod.  For example:
```
apiVersion: v1
kind: Pod
metadata:
  name: promtest
  annotations:
    prometheus.io/scrape: "true"
    prometheus.io/port: "9101"
spec:
  containers:
  - image: busybox
    name: bb
    command: [ "sleep", "86400" ]
```

Reply:
I tried before without the services and prometheus was not scraping the metrics....